web/templates/login.html

```diff
@@ -0,0 +1,65 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+
+{% block content %}
+
+<style>
+	body {
```

Comment:
also when you transfer this, don't override the body tag unless you actually want the styles to be global